test/check-multi-machine

```diff
@@ -405,25 +406,27 @@ class TestMultiMachine(MachineCase):
         b.logout()
         b.wait_visible("#login")
 
-        m1.execute("semanage port -a -t ssh_port_t -p tcp 2222")
-        m2.change_ssh_port(2222)
-        self.login_and_go(None)
-        b.go(machine_path)
-        b.wait_visible(".curtains div.spinner")
-        b.wait_visible("#machine-reconnect")
-        start_machine_troubleshoot(b)
-        b.wait_in_text('#troubleshoot-dialog h4', "Could not contact")
-        b.wait_present("#edit-machine-port")
-        b.set_val("#edit-machine-port", "2222")
-        b.click('#troubleshoot-dialog .btn-primary')
-        b.wait_in_text('#troubleshoot-dialog h4', "Log in to")
-        b.wait_visible("#login-diff-password")
-        b.set_val("#login-custom-password", "alt-password")
-        b.click('#troubleshoot-dialog .btn-primary')
-        b.wait_popdown('troubleshoot-dialog')
+        # https://bugzilla.redhat.com/show_bug.cgi?id=1290659
```

Comment:
Perhaps this should be a known issue with fedora-atomic. It's something that actually affects users (every time they try to perform the task of changing the ssh port), and something Cockpit cannot work around.